enterprise_subsidy/apps/api/v1/tests/test_views.py

```diff
@@ -216,6 +216,7 @@ def test_get_one_subsidy(self):
             "internal_only": False,
             "revenue_category": RevenueCategoryChoices.BULK_ENROLLMENT_PREPAY,
             "is_active": True,
+            "total_deposits": self.subsidy_1.starting_balance,
```

Comment:
if you can, try to add either test data or another test case where an Adjustment record(s) exist(s).

Reply:
Opted to create a new test with new test data simulating both a positive and negative adjustments, validating that the total deposits amount did in fact change after the adjustments were made.